src/backend/looped/world/groundsnow.cpp

```diff
@@ -0,0 +1,25 @@
+#include "backend/looped_command.hpp"
+#include "fiber_pool.hpp"
+#include "gta_util.hpp"
+#include "script.hpp"
+#include "services/tunables/tunables_service.hpp"
+
+namespace big
+{
+	class ground_snow : looped_command
+	{
+		using looped_command::looped_command;
+
+		virtual void on_tick() override
+		{
+			g_tunables_service->set_tunable<bool>(RAGE_JOAAT("TURN_SNOW_ON_OFF"), true);
```

Comment:
Do not change this, but just for reference, each Tunable/Global/Local/whatever is always 4 bytes wide. It is generally good practice to type it as BOOL not bool, as BOOL is the C-style boolean, and will force the compiler to write 4 bytes, whereas bool will only force it to write 1 byte. Assuming a 0-contamination environment, there should never be a scenario where the upper 3 bytes of this are written to, but the script VM only types things as 4 bytes wide, there is no actual type of bool to the script VM.

That being said, I am the only one of the YimMenu contributors who does this, as:
A: I learned on C a million years ago, never gave it up.
B: I always program the cheat to use whatever the game is using, an old habit from the C64 days as poking outside the memory space of what you were given always resulted in a HCF scenario.

You are free to ignore this.

Reply:
Always open to new information. I'll keep it in mind! :)

Reply:
> I believe the local weather changer enables snow automatically

It doesn't. The weather "snow" or "snowlight" (can't remember what its called) just enables the snowing effect. Ground snow has to be toggled through the tunable.

Reply:
> It's [supposed to](https://github.com/YimMenu/YimMenu/blob/master/src/backend/looped/world/weather_override.cpp#L31), though it's been a while since I've used this menu

Oh never noticed that, although when I tried changing the weathers ingame nothing happened. Anyhow the option allows the user to enable ground snow whenever, not just with the snow weather.